src/whining.js

```diff
@@ -0,0 +1,68 @@
+// Description:
+//   Whining
+//
+//   A hubot script that creates an agenda for Wednesday Whining
+//
+// Configuration:
+//   HUBOT_IDIO_WHINING_BRANCH: optional
+//   HUBOT_IDIO_WHINING_ORG: optional
+//   HUBOT_IDIO_WHINING_REPO: optional
+//   HUBOT_IDIO_WHINING_TOKEN: required
+//
+// Commands:
+//   hubot it's whine time - Push a whining agenda to the docs repo
+//
+// Author:
+//   Mal Graty
+
+'use strict'
+
+const Octokat = require('octokat')
+const octo = new Octokat({token: process.env.HUBOT_IDIO_WHINING_TOKEN})
+
+const branch = process.env.HUBOT_IDIO_WHINING_BRANCH || 'master'
```

Comment:
I thought we addressed this in the whining session, but it could also be me remembering my own thoughts rather than vocalised discussion, so here's my thinking:

An issue being postponed is not the same as it never being on the agenda. So it just means that postponed issues will appear on the agenda for mutltiple weeks. Not the end of the world. If more wide ranging changes are required then a normal PR cycle can be followed on top of the base agenda, but for the most part going straight to master with the base version will mean less overhead in the vast majority of cases.

If it turns out that that assertion is incorrect as we go forward, we can revisit the decision.